app/src/main/res/drawable/ic_select_quality.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file is not consistent with the other icon files. See `ic_guide.xml` for example:

- pathData should be on a single line (Android Studio formatter should do this automatically)
- `tint` attribute needs to be added
- fill color should be `@android:color/white`